drivers/adc/adc_ads1119.c

```diff
@@ -22,10 +22,10 @@ LOG_MODULE_REGISTER(ADS1119, CONFIG_ADC_LOG_LEVEL);
 
 
 #define ADS1119_CONFIG_VREF(x) ((x) & BIT(0))
-#define ADS1119_CONFIG_CM(x) ((x) & BIT(1))
-#define ADS1119_CONFIG_DR(x) ((x) & (BIT_MASK(2) << 2))
-#define ADS1119_CONFIG_GAIN(x) ((x) & BIT(4))
-#define ADS1119_CONFIG_MUX(x) ((x) & (BIT_MASK(3) << 5))
+#define ADS1119_CONFIG_CM(x) (((x) & BIT(0)) << 1)
+#define ADS1119_CONFIG_DR(x) (((x) & BIT_MASK(2)) << 2)
+#define ADS1119_CONFIG_GAIN(x) (((x) & BIT(0)) << 4)
+#define ADS1119_CONFIG_MUX(x) (((x) & BIT_MASK(3)) << 5)
```

Comment:
perhaps use FIELD_PREP? https://docs.zephyrproject.org/latest/doxygen/html/group__sys-util.html#gaa03c8b31bf67a097dd9f8153a04ef86b

Reply:
Changed in [118ab41](https://github.com/zephyrproject-rtos/zephyr/pull/70120/commits/118ab41b80b995bf1c6fe3805aa683617b20a636).